src/ssc_joystick_ros.cpp

```diff
@@ -507,6 +512,24 @@ int main(int argc, char **argv)
 
   deceleration = deceleration_limit;
 
+  if (engage_speed_module || engage_steering_module)
+  {
+    std::cout <<"\nSPEED MODULE MODE: " << engage_speed_module <<"\nSTEERING MODULE MODE: " <<
+    engage_steering_module << std::endl;
+
+    if (engage_speed_module && engage_steering_module)
+    {
+      std::cout <<"\nSPEED AND STEERING CONTROL SET TO ENGAGE" << std::endl;
+    }
+
+    engaged = 1;
+  }
+  else
+  {
+    std::cout << "\nNO MODULE HAS BEEN SET TO ENGAGE, SSC WILL NOT BE ACTIVE" << std::endl;
+    engaged = 0;
```

Comment:
You don't want to set engaged here, that just happens when they press the buttons.

Should probably test this on a vehicle.

Reply:
Yup, I wanted to hold off on new commits till testing this updated version.